jda/src/main/java/co/aikar/commands/JDACommandPermissionResolver.java

```diff
@@ -0,0 +1,36 @@
+package co.aikar.commands;
+
+
+import net.dv8tion.jda.core.Permission;
+import net.dv8tion.jda.core.entities.Member;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class JDACommandPermissionResolver implements CommandPermissionResolver {
+    private Map<String, Integer> discordPermissionOffsets;
+
+    public JDACommandPermissionResolver() {
+        discordPermissionOffsets = new HashMap<>();
+        for (Permission permission : Permission.values()) {
+            discordPermissionOffsets.put(permission.name().toLowerCase().replaceAll("_", "-"), permission.getOffset());
+        }
+    }
+
+    @Override
+    public boolean hasPermission(JDACommandEvent event, String permission) {
+        Member guildMember = event.getIssuer().getMember();
+        if (guildMember == null) {
```

Comment:
bot owner should prob return true here

Reply:
this would ensure permission based commands still work in PM.

Reply:
Good idea